charts/kube-green/templates/certificate.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: kube-green-serving-cert
+  namespace: kube-green
```

Comment:
Can't we put `namespace: {{ .Release.Namespace }}` instead or remove the parameter namespace

Reply:
Added namespace customizations and removed the lock from the kube-green namespace as you commented. Thks friend!